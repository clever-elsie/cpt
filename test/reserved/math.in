\print(
  \log(10),
  \log_{2}(3),
  \log10(10),
  \abs(-2),
  \abs(-4.0),
  \cos(1),
  \sin(1),
  \tan(1),
  \acos(1),
  \asin(1),
  \atan(1),
  \cosh(1),
  \sinh(1),
  \tanh(1),
  \ceil(1.5),
  \floor(1.5),
  \round(1.5),
  \trunc(1.5)
)