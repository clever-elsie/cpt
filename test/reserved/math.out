2.30259
1.58496
1
2
4
0.540302
0.841471
1.55741
0
1.5708
0.785398
1.54308
1.1752
0.761594
2
1
2
1
18
